common/src/parse.cpp

```diff
@@ -44,7 +44,11 @@
 
 #include <pcl/console/parse.h>
 #include <pcl/console/print.h>
+
 #include <boost/algorithm/string.hpp>
+#include <boost/mpl/deref.hpp>
+#include <boost/mpl/find_if.hpp>
+#include <boost/mpl/vector.hpp>
```

Comment:
Arent' the goal eventually to get free from boost :) ?

Reply:
:laughing: I don't think that's ever feasible. I did however reduce the scope and removed the usage of MPL which might not be familiar to many readers